doc/content/gateways/moko-mkgw2-lw/_index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "MOKOSMART MKGW2-LW LoRaWAN Gateway"
```

Comment:
Their webpage says MOKOSmart

Reply:
@Fakerallen wrote it so I'm going to leave it